src/types/network.ts

```diff
@@ -30,4 +31,7 @@ export interface VaultService {
         vaultAddress: string,
         strategyAddress: string
     ) => Promise<StrategyMetaData>;
+    getStrategyReports(
```

Comment:
Add this function to the `VaultService` interface so that retrieving strat reports is now encapsulated in the service rather then in a util function